src/components/map-components/google-map.js

```diff
@@ -73,6 +76,24 @@ class GoogleMap extends Component {
       latitude = get(resp, 'data.latitude');
       longitude = get(resp, 'data.longitude');
       selection = resp.data;
+    } else if (state) {
+      // state in url
+
+      // check if its actually a state, it will change the necessary search term
+      const searchTerm = (NON_STATES.includes(state.toLowerCase())) ? state : `state of ${state}`;
+      const results = await geocodeByAddress(searchTerm)
```

Comment:
worried about empty results error here... can we catch and return an empty array?

Reply:
the isError will catch it actually, you can try it out by doing /map/asdasdasasdasd

Reply:
![image](https://user-images.githubusercontent.com/3998637/83316528-4c27f280-a1f4-11ea-9f89-3592fe6da2e4.png)

Reply:
![image](https://user-images.githubusercontent.com/3998637/83316539-59dd7800-a1f4-11ea-8c5b-007f037e662a.png)